main.go

```diff
@@ -380,6 +381,40 @@ var idCmd = &cobra.Command{
 	},
 }
 
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "shows current version of bidbot",
+	Args:  cobra.ExactArgs(0),
+	Run: func(c *cobra.Command, args []string) {
+		local := buildinfo.Summary()
```

Comment:
It's longer than expected because I want to:
1. print local version even if there's problem getting remote version.
2. if local and remote version matches, only print once.
3. print both if they don't match.

Usually local and remote and the same, but just in case if someone installs it on their laptop pointing to a deamon running on the mining server...